main.py

```diff
@@ -0,0 +1,35 @@
+from funk import searchingGame, mathematicOper
+import time
+
+
+print("Hello, my name is Tom, let's play")
+time.sleep(1)
+print("Enter number of game you want:")
+print("1)  Guess number")
+print("2)  Calculate something(+,-,/,*)")
+print("3)  STOP the game  ")
```

Comment:
This duplicates code fragment below. Therefore, I recommend to wrap it up in a function like:

```python
def print_main_menu():
    print("Enter number of game you want:")
    print("1)  Guess number")
    print("2)  Calculate something(+,-,/,*)")
    print("3)  STOP the game  ")
```